projects/jdwp/defs/command_sets/event_request.py

```diff
@@ -0,0 +1,130 @@
+from projects.jdwp.defs.schema import (
+    Command,
+    Field,
+    Struct,
+    CommandSet,
+    IntegralType,
+    Type,
+)
+from projects.jdwp.defs.constants import ErrorType
+from typing import Union
+
+
+CountModifier = Struct([
+    Field("count", IntegralType.INT, "Count before event. One for one-off.")
+])
+
+ConditionalModifier = Struct([
+    Field("exprID", IntegralType.INT, "For the future")
+])
+
+ThreadOnlyModifier = Struct([
+    Field("thread", Type.THREAD_ID, "Required thread")
+])
+
+ClassOnlyModifier = Struct([
+    Field("class_", Type.CLASS_ID, "Required reference type")
+])
+
+ClassMatchModifier = Struct([
+    Field("classPattern", Type.STRING, "Restricted regular expression")
+])
+
+StepModifier = Struct([
+    Field("thread", Type.THREAD_ID, "Required thread"),
+    Field("size", IntegralType.INT, "Size of each step"),
+    Field("depth", IntegralType.INT, "Relative call stack limit")
+])
+
+
+ClassExcludeModifier = Struct([
+    Field("classPattern", Type.STRING, "Restricted regular expression")
+])
+
+LocationOnlyModifier = Struct([
+    Field("loc", Type.LOCATION, "Required location")
+])
+
+ExceptionOnlyModifier = Struct([
+    Field("exceptionOrNull", Type.CLASS_ID, "Optional reference type"),
+    Field("caught", Type.BOOLEAN, "True if exception was caught"),
+    Field("uncaught", Type.BOOLEAN, "True if exception was uncaught")
+])
+
+FieldOnlyModifier = Struct([
+    Field("declaring", Type.CLASS_ID, "Required reference type"),
+    Field("fieldID", IntegralType.INT, "Required field ID")
+])
+
+InstanceOnlyModifier = Struct([
+    Field("instance", Type.OBJECT_ID, "Required object")
+])
+
+SourceNameMatchModifier = Struct([
+    Field("sourceNamePattern", Type.STRING, "Restricted regular expression")
+])
+
+
+__Modifier = Union[
+        CountModifier,
+        ConditionalModifier,
+        ThreadOnlyModifier,
+        ClassOnlyModifier,
+        ClassMatchModifier,
+        ClassExcludeModifier,
+        LocationOnlyModifier,
+        ExceptionOnlyModifier,
+        FieldOnlyModifier,
+        SourceNameMatchModifier,
+    ]
+
+__SetCommand_outArray_Element = Struct(
+    [
+        Field("modKind", IntegralType.BYTE, "Modifier kind"),
+        Field("modifier", __Modifier, "Modifier"),
+    ]
+)
+
+__SetCommand_out = Struct(
+    [
+        Field("eventKind", IntegralType.BYTE, "The kind of event to request."),
+        Field("suspendPolicy", IntegralType.BYTE, "The suspend policy to use."),
+       Field(
+            "modifiers",
+            IntegralType.INT,
+            "Constraints used to control the number of generated events.",
+        ),
+        Field("outModifiers", __SetCommand_outArray_Element, "Modifier array."),
```

Comment:
The `modifiers` field needs to be of type ArrayLength and `outModifiers` needs to be of an array type